### ISSUE TITLE
CSS Variables are ignored

### ISSUE THREAD

**Issue by mpahenrique** [kristerkari/react-native-stylus-transformer#6]
Hello. If I use the code above in my stylus file, react native does not understand the variable value, and my "test" property does not work.

```stylus
:root {
    --test : red
}

.test
    background-color: var(--test);
```

**Comment by kristerkari** [comment]
Yes, CSS variables are not supported because they would require a runtime library to be used.

It is however possible to implement. Here's the issue to track adding support for CSS custom properties/variables:
https://github.com/kristerkari/react-native-css-modules/issues/10

**Comment by kristerkari** [comment]
For now you can use Stylus' variables instead of CSS variables.

**Comment by kristerkari** [comment]
I released a new version 1.2.0 with a new `renderToCSS` method that can be used together with PostCSS to add support for CSS variables.

Please have a look at the README file for instructions:
https://github.com/kristerkari/react-native-stylus-transformer#css-custom-properties-css-variables
